tests/conftest.py

```diff
@@ -149,3 +149,24 @@ def create_file_structure(root_path: Path, file_structure_schema: FileStructureS
             new_root_path = root_path / Path(path_str)
             new_root_path.mkdir()
             create_file_structure(new_root_path, file_structure_schema=composite)
+
+
+@contextmanager
+def base_protostar_toml():
```

Comment:
also I wonder, shouldn't this be built into the `create_protostar_project` API?

Reply:
this contextmanager was moved to `run_cairo_test_runner` fixture which seems to resolve the issue